src/web/src/scan_run.ecpp

```diff
@@ -186,27 +188,24 @@ UserInfo user;
 
 
     if (rv == -1)
-        http_die ("internal-error", "mlmpool.get () failed.");
+        http_die ("internal-error", TRANSLATE_ME ("mlmpool.get () failed."));
     else
     if (rv == -2)
-        http_die ("internal-error", "client->sendto () failed.");
+        http_die ("internal-error", TRANSLATE_ME ("client->sendto () failed."));
     else
     if (rv == -3)
-        http_die ("internal-error", "client->recv () failed.");
+        http_die ("internal-error", TRANSLATE_ME ("client->recv () failed."));
     else
     if (rv == -4)
-        http_die ("request-param-required", "<range>");
+        http_die ("internal-error", TRANSLATE_ME ("fty-discovery returned error."));
```

Comment:
Sanity check: should this message have changed? Did the function change and/or was it wrong before?

Reply:
This call used to have a required 'range' parameter. Now it's supposed to be called without parameters, but error reporting was not updated to reflect that.
Therefore, in the first commit, I updated error reporting, and in the second commit, I added translation enablement.